tutorial/practicals/nemo/3_nemo_openmp/README.md

```diff
@@ -142,7 +142,7 @@ the number of MPI processes and resulting inter-process communication.)
    directives have been inserted. You should see that there are now
    `Directive` nodes in the PSyIR, e.g.:
 
-       14: OMPParallelDoDirective[]
+       14: OMPParallelDoDirective[omp_schedule=auto]
```

Comment:
The nemo part of the notebooks states the schedule is "static" not "auto". Could you please change this.

Reply:
Fixed the nemo notebook (well spotted).